resources/config.xml

```diff
@@ -209,7 +209,7 @@
         </niveles>
     </escenario>
     <credenciales>
-         <cantidadJugadores>2</cantidadJugadores>
+         <cantidadJugadores>1</cantidadJugadores>
```

Comment:
Try to not change these values, the default should be 2 (it's okay for now, the next person will return it to 2 but for the next branch)